core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -79,8 +79,8 @@ oppia.factory('UrlInterpolationService', [
         var INVALID_VARIABLE_REGEX = /(<{2,})(\w*)(>{2,})/;
 
         // Parameter values can only contain alphanumerical characters, spaces,
-        // hyphens, underscores, or periods.
-        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.])+$/;
+        // hyphens, underscores, periods or the equal to symbol.
+        var VALID_URL_PARAMETER_VALUE_REGEX = /^(\w| |_|-|[.]|=)+$/;
```

Comment:
That's a good find! I think that is the cause.

OK, so it's not a bug and we have to deal with it. Reading the function it looks like there's some escaping going on at the bottom. So, if you add '=' to this list, can you verify that it gets escaped correctly and that everything works?

If so, I retract my earlier statement, and it's fine to add '=' to the list. Also maybe append to the comment "Note that these parameter values are escaped before being added to the URL" and add a Karma test to ensure that that's the case when '=' signs are used.